pkg/pods/manifest.go

```diff
@@ -102,6 +103,21 @@ func ManifestFromBytes(bytes []byte) (*Manifest, error) {
 	return manifest, nil
 }
 
+func ManifestFromURI(manifestUri string) (*Manifest, error) {
+	// Fetch manifest (could be URI) into temp file
+	localMan, err := ioutil.TempFile("", "tempmanifest")
```

Comment:
hmm, well I think they'd only be left lying around if the `defer os.Remove` failed to run, right?